mrp_workcenter_workorder_link/mrp_view.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+
+<!-- mrp.workcenter -->
+
+<record id="mrp_workcenter_tree_view" model="ir.ui.view">
+    <field name="model">mrp.workcenter</field>
+    <field name="inherit_id" ref="mrp.mrp_workcenter_tree_view"/>
+    <field name="arch" type="xml">
+        <field name="name" position="before">
+            <button name="button_workcenter_line"
+                    string="Click to see workcenter's operations"
+                    icon="gtk-execute"
+                    type="object"/>
+        </field>
+    </field>
+</record>
+
+
+<record id="mrp_workcenter_view" model="ir.ui.view">
+    <field name="model">mrp.workcenter</field>
+    <field name="inherit_id" ref="mrp.mrp_workcenter_view"/>
+    <field name="arch" type="xml">
+        <xpath expr="//notebook/page[last()]" position="before">
+            <page string="Work Orders" name="workorder">
+                <group col="6">
+                    <button name="button_workcenter_line"
+                            help="Work Orders filtered with this workcenter"
+                            class="oe_link"
+                            icon="gtk-execute"
+                            type="object"/>
```

Comment:
I think it need a bigger button with a label instead of this little gear lose in the middle of the form.